src/dune_rpc_server/dune_rpc_server.mli

```diff
@@ -127,11 +115,15 @@ module Handler : sig
     -> unit
 
   module Private : sig
+    (** Raw long polling machinery only good for tests *)
+
+    type poller
+
     val implement_poll :
          's t
       -> 'a Procedures.Poll.t
-      -> on_poll:('s Session.t -> Poller.t -> 'a option Fiber.t)
-      -> on_cancel:('s Session.t -> Poller.t -> unit Fiber.t)
+      -> on_poll:('s Session.t -> poller -> 'a option Fiber.t)
+      -> on_cancel:('s Session.t -> poller -> unit Fiber.t)
```

Comment:
This looks odd. What can you do with the `poller` if it's abstract?

Reply:
Nothing. This function is only used for tests and the tests don't care about the poller argument. I could specialize this function for the test and omit this argument.

Reply:
Yeah, I think we should drop this argument then, it's pretty confusing.

Reply:
Done.